stable/artifactory/values.yaml

```diff
@@ -55,7 +55,7 @@ artifactory:
   image:
     # repository: "docker.bintray.io/jfrog/artifactory-oss"
     repository: "docker.bintray.io/jfrog/artifactory-pro"
-    version: 5.6.3
+    version:
```

Comment:
A version should be listed to pin the version for reproducible deployments

Reply:
I was actually thinking that having the Chart.yaml's `appVersion` as a single source of version covers it.
I have a template in the '_helpers.tpl' called "artifactory.artifactory.version", which is used in the artifactory-deployment.yaml:
`image: '{{ .Values.artifactory.image.repository }}:{{ template "artifactory.artifactory.version" . }}'`
This way, I don't need to change the version in multiple files.
Did I miss something?

Reply:
I didn't see that. I like it.

Prior to merging this I would like to have a quick chat with the other maintainers.

/hold
/assign

Reply:
/hold cancel

@eldada we had a conversation about this in the charts chat call today. Several maintainers weighed in.

What you've done with the helpers is interesting and on more complicated charts, with multiple images using the same version, could make updates easier. This does make updating the chart easier for maintainers.

But, for those consuming the chart it makes it a little more difficult. What version being used isn't in the README. How you can set it yourself isn't intuitive. I've already witnessed a few people trip over setting their own tag version.

Because we want to prioritize chart consumers and their experience we would prefer to have explicit versions in the values.yaml file instead of pulling a default from the Chart.yaml file. We understand it's a little more work for chart maintainers and there's some duplication. But, it appears to be more intuitive for consumers.

Reply:
@mattfarina - I understand. I will revert this to the old format.
I think there should be an effort towards a best practice of a single source for the app version, that is used across all chart. I'll let you and the team play with this idea.
Maybe it's just an issue of better documentation (?)